open-craft/edx-platform
# [SE-3401] Upgrades Webpack to v4.44.2 to support content hashing

Upgrades Webpack from v2.7.0 to v4.44.2 so that content hashing is possible.

**JIRA tickets**: SE-3401

**Sandbox URL**: 
- **LMS**: https://pr265.sandbox.stage.opencraft.hosting/
- **Studio**: https://studio.pr265.sandbox.stage.opencraft.hosting/

**Testing instructions**:

1. Spawn multiple instances and add to load-balancer. 
1. Open the Developer console for the LMS and Studio and verify there are no 404 messages for `commons.js`.
1. SSH into the different instances and verify that the output for `ls /edx/var/edxapp/staticfiles/studio/bundles/ | grep -i commons` is the same among the different instances.
1. Compare the files in `/edx/var/edxapp/staticfiles/studio/bundles/` and `/edx/var/edxapp/staticfiles/bundles/` between both App Servers and make sure they have the same filename (including hashes).
1. Compare the files in `/edx/var/edxapp/staticfiles/studio/bundles/` and `/edx/var/edxapp/staticfiles/bundles/` between app servers with old instances (without the fix) and new ones (with the fix) and make sure everything is similar/acting normal. 
1. Verify that the assets are getting loaded correctly.

**Reviewers**
